table/queries/merged_reduced_scans.sql

```diff
@@ -21,6 +21,152 @@ CREATE TEMP FUNCTION CleanError(error STRING) AS (
     "port\\.[0-9]+", "port.[PORT]")
 );
 
+# Classify errors we believe occurred during
+# the TCP initialization phase of the probe.
+# Input is a string returned from CleanError
+# Output is a small set of classification strings.
+CREATE TEMP FUNCTION ClassifyDialError(error STRING) AS (
+  IF(error = "address already in use", "INVALID",
```

Comment:
Let's use CASE..WHEN instead: https://cloud.google.com/bigquery/docs/reference/standard-sql/conditional_expressions#case_expr

Reply:
switched to case/when